.github/script.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+app_name=$1
+image=$2
+namespace=$3
+container_name=$4
+
+deploy=`oc get deployment $app_name -n $namespace`
+if [[ "$?" -eq 0 ]]; then
+    oc set image deployment/$app_name $container_name=$image -n $namespace
+    oc rollout restart deployment/$app_name -n $namespace
+else
+    oc new-app $image --name $app_name -n $namespace
```

Comment:
I don't believe that a deploment with `oc new-app` would be useful? The ui can't run without the api?

Reply:
agree, i'll remove that part